src/test/java/org/broadinstitute/hellbender/tools/SplitReadsIntegrationTest.java

```diff
@@ -139,7 +139,8 @@ private int getReadCounts(final Path tempDirectory, final String baseName, final
                 .filter(f -> f.getType().isAssignableFrom(SamReader.Type.class))
                 .map(f -> orNull(() -> f.get(null)))
                 .filter(v -> v instanceof SamReader.Type)
-                .map(v -> (SamReader.Type) v);
+                .map(v -> (SamReader.Type) v)
+                .filter(v -> !v.fileExtension().equals("sra"));
```

Comment:
Add a comment here explaining that we're consciously excluding sra from these tests.